.ado/windows-vs-pr.yml

```diff
@@ -628,6 +628,13 @@ jobs:
         submodules: false # set to 'true' for a single level of submodules or 'recursive' to get submodules of submodules
         persistCredentials: false # set to 'true' to leave the OAuth token in the Git config after the initial fetch
 
+      # Run unbroken before installing packages to avoid scanning node_modules
+      # The exclusions of unbroken don't support wildcards yet and we have many throughout the repo
+      - task: CmdLine@2
```

Comment:
I like running this during PR, but if we have it part of VS PR validation, it will become a required check that can break if external links change.

The docs site runs unbroken separately from Azure Pipelines. I wonder if we can do that here, then just not mark that as a required check. We're already triggering Unbroken on merge, so I'm wondering if we just have the wrong trigger specified.

Reply:
Agree, I'll see if 8 can make it emit warnings